eng/pipelines/common/templates/steps/ensure-dotnet-version.yml

```diff
@@ -0,0 +1,77 @@
+#################################################################################
+# Licensed to the .NET Foundation under one or more agreements.                 #
+# The .NET Foundation licenses this file to you under the MIT license.          #
+# See the LICENSE file in the project root for more information.                #
+#################################################################################
+parameters:
+- name: version
+  type: number
+  default: 9
+- name: sdk
```

Comment:
Why not just have this as a string with values of 'sdk' and 'runtime' - that way when it's passed into the later tasks, we don't have to have if expressions